.github/workflows/tests.yml

```diff
@@ -35,6 +35,14 @@ jobs:
           git clone https://github.com/qiboteam/qibolab
           cd qibolab
           poetry install --no-interaction --with tests --all-extras
+          cd ..
+          git clone https://github.com/qiboteam/qibolab-qm
+          cd qibolab-qm
+          poetry install --no-interaction --with tests --all-extras
+          cd ..
+          git clone https://github.com/qiboteam/qibolab-qblox
+          cd qibolab-qblox
+          poetry install --no-interaction --with tests --all-extras
```

Comment:
Given the standardized structure, maybe we can replace this with a bash `for` loop.

Reply:
bb8bb477e855c241e3c811d8d622454f3055478c, 9e7690146321878c5a6848734428ba80ed13e763